service/src/main/java/bio/terra/workspace/service/workspace/model/WsmObjectType.java

```diff
@@ -0,0 +1,10 @@
+package bio.terra.workspace.service.workspace.model;
+
+public enum WsmObjectType {
+  UNKNOW,
```

Comment:
This is mispelled.

// This row was written before change_subject_type column was added
UNKNOWN,

But as I said in another comment, if something isn't set, we usually do null instead of "unknown".

Reply:
why? i tend to avoid nullable fields just because it's easy to forget checking null

Reply:
If I see

```
public method(@Nullable WsmObjectType wsmObjectType) {
```

I know that `wsmObjectType` might be null, and I'll check for it.

If I see

```
public method(WsmObjectType wsmObjectType) {
```

For most enums, this means wsmObjectType is always set. I have to know that for this particular enum "unset means UNKNOWN". We have many enums and this is the only one that has UNKNOWN. I think it's more likely I'll remember the @Nullable case, than the UNKNOWN case. :)

Reply:
My main interaction with enums were from proto where the default is always the first enum value so I always start with an UNKNOWN. In this case, it doesn't have an apparent benefit.

OTOH, UNKNOWN.name() is still UNKNOWN which i think is a sensible output though. I will change it to null to be consistent